scriptworker/test/test_gpg.py

```diff
@@ -692,7 +599,15 @@ def test_verify_signed_git_commit_output_exception(output):
 
 
 @pytest.mark.asyncio
-async def test_get_git_revision_exception(mocker):
+async def test_get_latest_tag():
+    parent_dir = os.path.dirname(__file__)
+    expected = subprocess.check_output(["git", "describe", "--abbrev=0"], cwd=parent_dir)
```

Comment:
Do we have a way to systematically know the tag names and the number of tags in the test suite? Like either create a new repo and add two tags and see which one comes up?